nitro-protocol/contracts/interfaces/IForceMoveApp.sol

```diff
@@ -10,7 +10,7 @@ import './INitroTypes.sol';
 interface IForceMoveApp is INitroTypes {
     /**
      * @notice Encodes application-specific rules for a particular ForceMove-compliant state channel. Must revert when invalid support proof and a candidate are supplied.
-     * @dev Encodes application-specific rules for a particular ForceMove-compliant state channel. Must revert when invalid support proof and a candidate are supplied.
+     * @dev Depending on application, it might be useful to stricken state mutability of an implementation to 'pure' for security reasons.
```

Comment:
```suggestion
     * @dev Depending on the application, it might be desirable to narrow the state mutability of an implementation to 'pure' to make security analysis easier. 
```